src/Validation.ContentScan.Core/CheckContentScanData.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using NuGet.Jobs.Validation.ContentScan;
+
+namespace Validation.ContentScan.Core
+{
+    public class CheckContentScanData
+    {
+        public static CheckContentScanData NewStartContentScanData(
+           Guid validationTrackingId,
+           Uri contentUrl)
+        {
+            return new CheckContentScanData(
+                ContentScanOperationType.StartScan,
+                startContentScan: new StartContentScanData(
+                    validationTrackingId,
+                    contentUrl),
+                checkContentScanStatus: null,
+                deliveryCount: 0);
+        }
+
+        public static CheckContentScanData NewCheckContentScanStatus(
+          Guid validationTrackingId)
+        {
+            return new CheckContentScanData(
+                ContentScanOperationType.StartScan,
```

Comment:
Should this be `ContentScanOperationType.CheckStatus`? 

```suggestion
                ContentScanOperationType.CheckStatus,
```

Reply:
yes, copy paste mistake :)